protobuf/go/flow/access/access.pb.go

```diff
@@ -1,2577 +1,1570 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.17.1
 // source: flow/access/access.proto
 
 package access
 
 import (
+	fmt "fmt"
+	proto "github.com/golang/protobuf/proto"
```

Comment:
so `buf` seems to be reverting to the older protobuf implementation for golang - 

This module (github.com/golang/protobuf) contains Go bindings for protocol buffers.

see [here](https://github.com/golang/protobuf#go-support-for-protocol-buffers) - 

> It has been superseded by the google.golang.org/protobuf module, which contains an updated and simplified API, support for protobuf reflection, and many other improvements. We recommend that new code use the google.golang.org/protobuf module.

Reply:
nevermind..we are currently using the old `github.com/golang/protobuf/proto` as well.